pkg/baremetalset/const.go

```diff
@@ -20,9 +20,6 @@ const (
 	// AppLabel -
 	AppLabel = "osp-baremetalset"
 
-	// BaremetalHostRemovalAnnotation - Annotation key placed on BaremetalHost resources to target them for OpenStackBaremetalSet scale-down
-	BaremetalHostRemovalAnnotation = "osp-director.openstack.org/baremetalset-delete-baremetalhost"
-
```

Comment:
This is old and not used.  We use this [1] instead.

[1] https://github.com/openstack-k8s-operators/osp-director-operator/blob/b5d119de2c7998c03061cdf1d8f5ea1ccf9d4849/api/shared/annotations.go#L4-L5